apps/web/src/pages/settings/tabs/InAppCenterCard.tsx

```diff
@@ -39,7 +39,7 @@ export const InAppCenterCard = () => {
                 borderRadius: '7px',
               },
             })}
-            language="jsx"
+            language="javascript"
```

Comment:
```suggestion
            language="html"
```

First of all, great catch! it is indeed not a `jsx`, but I think that the more appropriate tag for this would be HTML. Since the embed variable that is displayed by Prism on line 45 is an HTML snippet. 

What do you think about it?

Reply:
Yes but we need to add **html**  language in
novu\node_modules\.pnpm\prism-react-renderer@1.3.1_react@17.0.2\node_modules\prism-react-renderer\index.d.ts